c/client.h

```diff
@@ -0,0 +1,19 @@
+#include <stdint.h>
+
+#ifndef CLIENT_INCLUDED
+#define CLIENT_INCLUDED
```

Comment:
@ewanas can you explain how this is used?

Reply:
The prevents the file from being re-included. The `#endif` must be placed after the end of the file.